### ISSUE TITLE
Facet's highlighting 

### ISSUE THREAD

**Issue by jrpatterson** [psu-libraries/etda_explore#43]
The WIP Shading should be adjusted.
Current:
 ![current-preferred-Screenshot 2022-06-17 at 14-34-20 ETDA](https://user-images.githubusercontent.com/2422371/174381307-0150ff8e-18ee-445c-b6c5-1fb21dca2a8b.png)

WIP Version

![Screenshot 2022-06-17 at 14-34-10 eTD Explore](https://user-images.githubusercontent.com/2422371/174381340-6935444d-e48b-4470-871f-d58d419ea807.png)

**Comment by jrpatterson** [comment]
Scholarsphere handles this the same way as the WIP version so maybe keeping them the same and using what you have is more consistent.

**Comment by ajkiessl** [comment]
Yeah, I think this is just how the newer version of blacklight does it.
